src/Symfony/Bridge/Doctrine/Tests/Validator/Constraints/UniqueEntityValidatorTest.php

```diff
@@ -574,6 +574,11 @@ public function testValidateUniquenessWithCompositeObjectNoToStringIdEntity()
 
         $objectOne = new SingleIntIdNoToStringEntity(1, 'foo');
         $objectTwo = new SingleIntIdNoToStringEntity(2, 'bar');
+
+        $this->em->persist($objectOne);
```

Comment:
I think `CompositeObjectNoToStringIdEntity` cascades persist to `objectOne` and `objectTwo`? So not sure this will change anything?